src/container/internal.tsx

```diff
@@ -69,12 +69,15 @@ export default function InternalContainer({
    */
   useLayoutEffect(
```

Comment:
Can this issue be fixed by replacing `useLayoutEffect` with `useEffect`?

Reply:
Without having the condition it would behave the same.
I'm not 100% sure why `useLayoutEffect` have been used initially because .  I think changing that to `useEffect` does not impact the behaviour because we don't do any DOM readings or direct DOM manipulations inside the hook.

Reply:
Understood. More context: https://github.com/facebook/react/issues/21400

The issue still can be fixed by applying `useEffect` somewhere, but not this one.

In general, when you are using any effect, you are making a compromise causing expensive investigations and difficult bug fixes. Please read [this article](https://beta.reactjs.org/learn/you-might-not-need-an-effect) and avoid.